scss/_patterns_tooltips.scss

```diff
@@ -53,6 +53,16 @@ $triangle-height: $sp-unit;
       position: absolute;
       width: 0;
     }
+
+    &::after {
+      border-radius: 10% 90% 0% 100% / 100% 100% 0% 0%;
+      content: '';
+      height: 15px;
```

Comment:
I guess the 15px here is related to triangle height, so maybe we could reuse `$triangle-height` variable for that (and other positioning values)?